--- physx/include/foundation/PxSIMDHelpers.h
+++ physx/include/foundation/PxSIMDHelpers.h
@@ -48,7 +48,7 @@
 			const QuatV qV = V4LoadU(&q.x);
 			Vec3V column0V, column1V, column2V;
 			QuatGetMat33V(qV, column0V, column1V, column2V);
-#if defined(PX_SIMD_DISABLED) || (PX_LINUX && (PX_ARM || PX_A64))
+#if defined(PX_SIMD_DISABLED) || PX_ARM_FAMILY
 			V3StoreU(column0V, column0);
 			V3StoreU(column1V, column1);
 			V3StoreU(column2V, column2);
