--- CMakeLists.txt
+++ CMakeLists.txt
@@ -446,10 +446,10 @@
     if (UNIX AND NOT BASISU_BUILD_WASM)
         if (CMAKE_BUILD_TYPE STREQUAL Release)
             if (APPLE)
-                add_custom_command(TARGET basisu POST_BUILD COMMAND strip -X -x ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/basisu)
+                add_custom_command(TARGET basisu POST_BUILD COMMAND ${CMAKE_STRIP} -X -x ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/basisu)
                 #message("strip command: strip -X -x ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/basisu")
             else()
-                add_custom_command(TARGET basisu POST_BUILD COMMAND strip -g -X -x ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/basisu)
+                add_custom_command(TARGET basisu POST_BUILD COMMAND ${CMAKE_STRIP} -g -X -x ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/basisu)
                 #message("strip command: strip -g -X -x ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/basisu")
             endif()
         endif()
